src/sql/workbench/services/accountManagement/browser/media/accountListRenderer.css

```diff
@@ -3,12 +3,12 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.list-row.account-picker-list {
+.list-row.account-picker-list, .list-row.tenant-picker-list {
```

Comment:
Put these on separate lines - easier to see that they're different selectors

Reply:
And it's prooooobably fine but I really think these should be scoped down to be under the specific account list dropdown class in case something else decides to use a list with the same class names.